### PR TITLE
include an info response in the healthcheck endpoint

### PR DESCRIPTION
**Pull request by tnguyen14**
this addresses #64 

/cc @gergelyke

### CODE REVIEW

File: lib/terminus.js [hunk 1]
```diff
@@ -3,26 +3,29 @@
...
 }
 
-function sendSuccess (res) {
+function sendSuccess (res, info) {
   res.statusCode = 200
-  res.end(SUCCESS_RESPONSE)
+  if (info) {
+    SUCCESS_RESPONSE.info = info
+  }
+  res.end(JSON.stringify(SUCCESS_RESPONSE))
```
Comment (gergelyke) [inline lib/terminus.js]: if there is no `info` provided, can we used the cached JSON.stringify result, so it won't get stringified every time again?

**Comment by gergelyke** [comment]
looks great @tnguyen14 - just added a small perf-related comment

**Comment by tnguyen14** [comment]
Thanks! I've updated accordingly.

**Pull request merged**